pkg/notes/document/testdata/document_without_downloads.md.golden

```diff
@@ -0,0 +1,44 @@
+# Release notes for v1.16.1
+
+[Documentation](https://docs.k8s.io/docs/home)
+
+# Changelog since v1.16.0
+
+## Urgent Upgrade Notes 
+
+### (No, really, you MUST read this before you upgrade)
+
+ - If an API changes and no one documented it, did it really happen?
+ 
+## Changes by Kind
+
+### Other (Bug, Cleanup or Flake)
+ - This should definitely get you promoted.
+ 
+### API Change
+ - This might make people sad...or happy.
+ 
+### Bug
+ - This will likely get you promoted.
+ 
+### Cleanup
+ - This usually does not get you promoted but it should.
+ 
+### Deprecation
+ - Delorted.
+ 
+### Design
+ - Change the world.
+ 
+### Documentation
+ - There was  a library in Alexandria, Egypt once.
+ 
+### Failing Test
+ - Please run presubmit test!
+ 
+### Feature
+ - This will get you promoted.
+ 
+### Flake
+ - This *should* get you promoted.
+ 
```

Comment:
Can we somehow change the generation that there is a newline at the end of each file? Most editors will add that automatically which could lead into unwanted changes.

Reply:
Done.